k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -976,6 +1006,96 @@
     "sha256:e15927e26974af0e066018b056c5f51e514347950e164e4e5be822ed723ee7e0": ["v1.17.10"]
     "sha256:f5de4ad913ffd66ceee261f40fecaec852342168596c9e6d2d8c6fe9f1cd7359": ["v1.18.9"]
     "sha256:f86066539c4a8fb82715e049ef13c0161dbc3b534881b2315476b254b0556733": ["v1.19.1"]
+- name: mock/conformance
+  dmap:
+    "sha256:c9b97cfe2af7e54e8203fdc3fbc6e5151d32066cab5fd1e3c0a929c32b83af65": ["v1.19.3"]
```

Comment:
Okay this does not look correct. @puerco do you have an idea why those are listed here?

Reply:
@saschagrunert I am guessing this was done without using `krel` as these previously had to be manually removed :+1:

Reply:
need to remove the mock one as well